ansible_collections/arista/avd/roles/eos_designs/templates/designs/mpls-vpn/old_documentation/fabric-documentation.j2

```diff
@@ -0,0 +1,162 @@
+{% set fabric_switches = [] %}
```

Comment:
Please cleanup all these copies of the old files.

Reply:
Cleaned old files.